src/Uno.UI/UI/Xaml/Media/Animation/RepeatBehavior.cs

```diff
@@ -27,6 +28,11 @@ public RepeatBehavior(double count)
 
 		public RepeatBehavior(TimeSpan duration)
 		{
+			if (duration < TimeSpan.Zero)
+			{
+				throw new ArgumentOutOfRangeException(nameof(duration));
```

Comment:
nit nit: Not a blocker it could be added later but could we specify what's the expected range either in the Exception or as a Summary comment on the method? Someone passing `TimeSpan.Zero` might not know this is not a valid value.

Reply:
Zero wont actually throw here, at least not anymore in accordance with windows behavior, but only negative one will (hint: `<`).
Same for count, only negative, NaN, or infinity will throw.

These seems like a pretty reasonable and sensible constraints for count and duration.
But, yeah, more verbose error message is always nice.